### ISSUE TITLE
Remove linux-steamos

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#579]
I have gotten a lot of feedback from the community that the linux-steamos kernel is not needed/wanted. Let me explain the reasoning behind all of this.

We have run into various issues that are still outstanding:

- https://github.com/LukeShortCloud/winesapOS/issues/490
- https://github.com/LukeShortCloud/winesapOS/issues/532
- linux-steamos is very outdated and insecure

The latest upstream Linux kernel has all of the following features:

- Steam Deck controller support (will be available in Linux 6.3, soon to be released)
- Steam Deck sound driver
- AMD RDNA2 graphics driver support for the Steam Deck
- fsync
    - Although the fsync in linux-steamos has more functionality, we are considering shipping winesync which is a planned as a long-term replacement and is usable today: https://github.com/LukeShortCloud/winesapOS/issues/479
- All of the latest security updates

In summary, we will switch to the upstream Linux kernel instead of the one from SteamOS. We will still keep the Linux LTS kernel as our preferred and default kernel.

**Comment by LukeShortCloud** [comment]
winesync support has been reported as now working in Proton but the patches are not in upstream Proton yet.

**Comment by LukeShortCloud** [comment]
There is an unknown network patch in the Neptune kernel that Wine uses. It might be required for Easy Anti-Cheat to work (still requires confirmation).

Refer to: `https://github.com/ValveSoftware/Proton/issues/2907#issuecomment-1518722499`

**Comment by LukeShortCloud** [comment]
The last major feature that is in Linux SteamOS and not upstream yet is HDR support. There is also the Linux HDR kernel from the AUR for those interested in that feature. I will continue to maintain the Linux SteamOS package in the AUR.

**Comment by LukeShortCloud** [comment]
We will not remove linux-steamos from existing installations. Instead, new installations will not have the Linux SteamOS kernel by default. It can optionally be installed during the first-time setup.

I'm closing this in favor of this new GitHub Issue: https://github.com/LukeShortCloud/winesapOS/issues/620
